docs/features/preemption.md

```diff
@@ -0,0 +1,435 @@
+# Quickstart - Preemption
+
+Within the Gödel Scheduling System, Pod preemption emerges as a key feature designed to uphold optimal resource utilization.
+If a Pod cannot be scheduled, the Gödel scheduler tries to preempt (evict) lower priority Pods to make scheduling of the pending Pod possible.
+
+This is a Quickstart guide that will walk you through how preemption works in the Gödel Scheduling System.
+
+## Local Cluster Bootstrap & Installation
+
+Please refer to [Cluster Setup Guide](kind-cluster-setup.md) for creating a local KIND cluster installed with Gödel.
+
+## How Preemption Works
+
+When there is no more resource for scheduling a pending Pod, preemption comes into picture.
+Gödel scheduler tries to preempt (evict) lower priority Pods to make scheduling of the pending Pod possible, with a few of protection strategies being respected.
+
+### Quickstart Scenario
+
+To better illustrate the preemption features, let's assume there is one node with less than 4 CPU cores (actually 3.9 cores) available for scheduling.  
+
+```bash
+$ kubectl describe node kind-worker   
```

Comment:
I think it may be more convenient to also save a copy of the demonstrared yaml files under 'manifests/quickstart-feature-examples', like in the quickstart guide of basic pod scheduling.

Reply:
I actually think the opposite way. It would be more distracting. Users of those yamls must be readers of the quickstart docs. I expect users to stick with the same page while reading through this quickstart doc, instead of switching back and forth to check the content of yaml. Copy-and-paste is straightforward.

Reply:
in my recent PR, I did the same with Yue and created yaml files. I think that's more convenient.
but how about we do both?

Reply:
I kinda did both, actually.